twisted/internet/cfreactor.py

```diff
@@ -474,9 +478,9 @@ def install(runLoop=None, runner=None):
     @param runLoop: the run loop to use.
 
     @param runner: the function to call in order to actually invoke the main
-        loop.  This will default to L{CFRunLoopRun} if not specified.  However,
-        this is not an appropriate choice for GUI applications, as you need to
-        run NSApplicationMain (or something like it).  For example, to run the
+        loop.  This will default to L{CoreFoundation.CFRunLoopRun} if not specified.
+        However, this is not an appropriate choice for GUI applications, as you need
+        to run NSApplicationMain (or something like it).  For example, to run the
         Twisted mainloop in a PyObjC application, your C{main.py} should look
```

Comment:
these invalid references can be fixed by adding links to CoreFoundation and CFNetwork